### ISSUE TITLE
Support Babe

### ISSUE THREAD

**Issue by bkchr** [paritytech/cumulus#116]
We should support Babe for Block authoring.

Blocked on: https://github.com/paritytech/cumulus/issues/36

**Comment by bkchr** [comment]
Related: https://github.com/paritytech/cumulus/issues/72

**Comment by bkchr** [comment]
This will probably not come for quite some time.

**Comment by wischli** [comment]
> It will not be done.

This implies parachains won't be able to use Babe as block production engine and need to stick to AuRa or a custom implementation, correct? I would be super interested in some arguments for this decision if you have the time. Thank you!

**Comment by xlc** [comment]
I guess we just need a new issue `support sassafras`?

**Comment by bkchr** [comment]
As @xlc already pointed out, Sassafras will be the one that will be supported. Not doing BABE is mainly a decision to not waste time. I mean when Sassafras comes, it will be superior to BABE. So, no one will probably switch to BABE knowing that Sassafras will come.

> I guess we just need a new issue `support sassafras`?

This will hopefully directly be part of the implementation of Sassafras, aka support for Parachains.
